src/main/java/ch/njol/skript/events/EvtPlayerArmorChange.java

```diff
@@ -0,0 +1,90 @@
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.registrations.Classes;
+import com.destroystokyo.paper.event.player.PlayerArmorChangeEvent;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.EquipmentSlot;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.concurrent.CompletionException;
+
+public class EvtPlayerArmorChange extends SkriptEvent {
+
+	private static boolean BODY_SLOT_EXISTS;
+
+	static {
+		if (Skript.classExists("com.destroystokyo.paper.event.player.PlayerArmorChangeEvent")) {
+			Skript.registerEvent("Armor Change", EvtPlayerArmorChange.class, PlayerArmorChangeEvent.class,
+				"[player] armo[u]r change[d] [(from|at) %-equipmentslots%]")
```

Comment:
I think this format is rather clunky. I'd rather see `on helmet change` or similar.